frontend/public/components/start-guide.tsx

```diff
@@ -70,9 +68,7 @@ export const withStartGuide = (WrappedComponent, disable: boolean = true) =>
               <OpenShiftGettingStarted canCreateProject={flags[FLAGS.CAN_CREATE_PROJECT]} />
             </HintBlock>
           </div>
-          {// Whitelist certain resource kinds that should not be disabled when no projects are
-          // available. Disabling should also be optional
-          !disable || WHITELIST.includes(kind) ? (
+          {!disable || !kindObj.namespaced || kind === RoleModel.kind ? (
```

Comment:
As per discussion with @spadgett, we should remove the special case for the Roles list page.
```suggestion
          {!disable || !kindObj.namespaced (
```